docs/source/guides/getting-started/running-idom.rst

```diff
@@ -34,17 +34,17 @@ will follow a pattern similar to the following:
     app = Application()
     configure(app, HelloWorld)
 
-You'll then run this ``app`` using a `ASGI <https://asgi.readthedocs.io/en/latest/>`__ or
-`WSGI <https://wsgi.readthedocs.io/>`__ server from the command line.
+You'll then run this ``app`` using an `ASGI <https://asgi.readthedocs.io/en/latest/>`__
+or `WSGI <https://wsgi.readthedocs.io/>`__ server from the command line.
 
 
 Running with `FastAPI <https://fastapi.tiangolo.com>`__
 .......................................................
 
 .. idom:: _examples/run_fastapi
 
-Then assuming you put this in ``main.py``, you can run the ``app`` using `Uvicorn
-<https://www.uvicorn.org/>`__:
+Then assuming you put this in ``main.py``, you can run the ``app`` using the `Uvicorn
```

Comment:
I personally would recommend advertising `Hypercorn` instead of `Uvicorn`
`Uvicorn` has Windows compatibility issues that have existed for a long time now.

Reply:
FastAPI's docs currently use uvicorn in their examples. Not really sure why, but I figure we should match what they are suggesting.